platform/o.n.swing.laf.flatlaf/src/org/netbeans/swing/laf/flatlaf/fontscolors/FlatLafDark-highlights.xml

```diff
@@ -49,4 +49,8 @@
     <fontcolor name="text-limit-line-color" foreColor="ff555555"/>
     <fontcolor name="trailing-whitespace"/>
     <fontcolor name="west-sidebars-color"/>
+    <fontcolor name="coverage-covered" bgColor="005000"/>
+    <fontcolor name="coverage-uncovered" bgColor="404040"/>
```

Comment:
This one is a neutral grey, but it should be some variant of red. Perhaps try 813438 instead.

Reply:
![image](https://user-images.githubusercontent.com/1381701/72489715-c74eb600-37c9-11ea-8d7d-f45c80365575.png)

Reply:
Not bad after all. Thanks.

Reply:
![image](https://user-images.githubusercontent.com/1381701/72489826-3cba8680-37ca-11ea-8f29-428d4f76f964.png)